src/RestApiClient.js

```diff
@@ -133,7 +133,12 @@ class RestApiClient {
     let urlParams = [];
 
     Object.keys(query).forEach((key) => {
-      urlParams.push(`${encodeURIComponent(key)}=${encodeURIComponent(query[key])}`);
+      if (key === 'filter') {
```

Comment:
It'd be nice to fix this for any nested properties, not just `filter`. This [StackOverflow answer](https://stackoverflow.com/a/42604801/811624) seems to take a good approach if you want to give it a try.